pkgs/development/python-modules/pre-commit-hook-ensure-sops/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, applyPatches, fetchFromGitHub, ruamel-yaml }:
+
+buildPythonPackage rec {
+  pname = "pre-commit-hook-ensure-sops";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "yuvipanda";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-8sMmHNzmYwOmHYSWoZ4rKb/2lKziFmT6ux+s+chd/Do=";
+  };
+
+  # Add the command-line entrypoint to pyproject.toml
+  patches = [ ././entrypoint.patch ];
```

Comment:
Please submit that patch upstream. It looks generally useful.

Reply:
Made a pull request here: https://github.com/yuvipanda/pre-commit-hook-ensure-sops/pull/11

Reply:
I have now used `fetchpatch` to remove the patch from the `nixpkgs` repository. Should I explicitly reference my fork with the patch or the `yuvipanda` main repository where the pull request is?

Reply:
@nialov Heads up that https://github.com/yuvipanda/pre-commit-hook-ensure-sops/pull/11 has been merged.